### ISSUE TITLE
Nested routes are not working when URL is refreshed

### ISSUE THREAD

**Issue by CreativeManix** [coryhouse/react-slingshot#65]
```
<Route path="/" component={App}>
    <IndexRoute component={FuelSavingsPage} />
    <Route path="about" component={AboutPage}/>
    <Route path="/auth"  component={AuthLayout} >
		<Route path="login" component={LoginPage}/>
		<Route path="atkn" component={AcceptTokenPage}/>
    </Route>
    <Route path="*" component={NotFoundPage} />
  </Route>
```

I have above configuration, when I navigate to /auth/login it works. however when I refresh the page, history fallback lib is not rendering the component instead it shows blank screen!

Should I do anything specifically?

**Comment by atfzl** [comment]
try this
```
<Route path="/" component={App}>
    <IndexRoute component={FuelSavingsPage} />
    <Route path="about" component={AboutPage}/>
    <Route path="auth" component={AuthLayout} >
        <IndexRoute component={LoginPage}/>
        <Route path="atkn" component={AcceptTokenPage}/>
    </Route>
    <Route path="*" component={NotFoundPage} />
  </Route>
```

**Comment by CreativeManix** [comment]
Sorry, It is not working. If I navigate using <Link> I am able to reach  _/auth/atkn_ however when I refresh my browser it is showing blank screen.

**Comment by atfzl** [comment]
Can you give the url to your repo.

**Comment by coryhouse** [comment]
@CreativeManix Get latest and try again. I added a leading slash in index.html for bundle.js. I believe this will fix your issue.

**Comment by coryhouse** [comment]
@CreativeManix - Is this issue resolved when you get latest?

**Comment by coryhouse** [comment]
Closing since this should have been fixed with a recent commit.

**Comment by jforaker** [comment]
This happens in the latest version as of 6/30/2016, and is undesirable...  any suggestions?  

The pull request above references a file that is no longer in the repo.

**Comment by coryhouse** [comment]
Great catch @jforaker. Bummer. This is my miss. The new structure for generating index.html via html-webpack-plugin caused this. I'm looking into a fix, but in the meantime, 4.0.0 will continue to work. I haven't cut a release in awhile so it's quite a few commits behind master.

**Comment by thatkit** [comment]
@CreativeManix I want to follow up on @coryhouse's answer on the leading slash solution.

In order to fix the issue by adding a leading slash to `<script src="index.js"></script>` (the react script imported in `index.html`).

`webpack.config.js`
```javascript
const config = {
  entry: './src/app/index.tsx',
  output: {
    filename: 'index.js',
    path: path.resolve(__dirname, 'dist'),
    publicPath: '/', // 👈 added this property
  },
  // other configs
```

When `publicPath` added, I got `<script src="/index.js"></script>` and solved the issue
